README.rst

```diff
@@ -0,0 +1,17 @@
+# go-ghe-email-alerts
```

Comment:
I guess you don't want the `# ` at the start of this line or the blank line below it?